firebase-perf/src/main/java/com/google/firebase/perf/session/SessionManager.java

```diff
@@ -49,8 +49,8 @@ public static SessionManager getInstance() {
 
   /** Returns the currently active PerfSession. */
   public final PerfSession perfSession() {
-    DebugEnforcementCheck.Companion.checkSession(
-        perfSession.isAqsReady, "Access perf session from manger without aqs ready");
+    FirebaseSessionsEnforcementCheck.Companion.checkSession(
```

Comment:
Add the `@JvmStatic` to this function too and you won't need to say .Companion in Java

Reply:
Done.